src/managers/messages.ts

```diff
@@ -24,24 +25,30 @@ export class MessagesManager extends BaseManager<MessagePayload, Message> {
     if (channel === undefined)
       channel = await this.client.channels.fetch(raw.channel_id)
 
-    let author = ((await this.client.users.get(
-      raw.author.id
-    )) as unknown) as User
+    let author = (await this.client.users.get(raw.author.id)) as unknown as User
 
     if (author === undefined) author = new User(this.client, raw.author)
 
-    const res = new this.DataType(this.client, raw, channel, author) as any
+    const res = new this.DataType(this.client, raw, channel, author)
     await res.mentions.fromPayload(raw)
     return res
   }
 
   async set(key: string, value: MessagePayload): Promise<any> {
```

Comment:
`set` can easily use `Promise<void>` or implicit return type because it does not return anything

Reply:
linter goes brrr

Reply:
then I'm quite curious why it is done somewhere else with another `set` method lol.
Eslint being stupid then I guess. Hopefully that can be changed soon

Reply:
oh wait, I misread

Reply:
fixed